src/Avalonia.Themes.Fluent/Slider.xaml

```diff
@@ -1,93 +1,256 @@
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="HorizontalDecreaseRect" Storyboard.TargetProperty="Fill">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="{DynamicResource SliderTrackValueFillPointerOver}" />
+          </ObjectAnimationUsingKeyFrames>
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="VerticalDecreaseRect" Storyboard.TargetProperty="Fill">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="{DynamicResource SliderTrackValueFillPointerOver}" />
+          </ObjectAnimationUsingKeyFrames>
+        </Storyboard>
+      </VisualState>
+
+    </VisualStateGroup>
+    <VisualStateGroup x:Name="FocusEngagementStates">
+      <VisualState x:Name="FocusDisengaged" />
+      <VisualState x:Name="FocusEngagedHorizontal">
+
+        <Storyboard>
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SliderContainer" Storyboard.TargetProperty="(Control.IsTemplateFocusTarget)">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="False" />
+          </ObjectAnimationUsingKeyFrames>
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="HorizontalThumb" Storyboard.TargetProperty="(Control.IsTemplateFocusTarget)">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="True" />
+          </ObjectAnimationUsingKeyFrames>
+        </Storyboard>
+      </VisualState>
+      <VisualState x:Name="FocusEngagedVertical">
+
+        <Storyboard>
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="SliderContainer" Storyboard.TargetProperty="(Control.IsTemplateFocusTarget)">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="False" />
+          </ObjectAnimationUsingKeyFrames>
+          <ObjectAnimationUsingKeyFrames Storyboard.TargetName="VerticalThumb" Storyboard.TargetProperty="(Control.IsTemplateFocusTarget)">
+            <DiscreteObjectKeyFrame KeyTime="0" Value="True" />
+          </ObjectAnimationUsingKeyFrames>
+        </Storyboard>
+      </VisualState>
+
+    </VisualStateGroup>
+
+  </VisualStateManager.VisualStateGroups>
+
+
+
+  <Style Selector="Slider">
+    <Setter Property="Background" Value="{DynamicResource SliderTrackFill}" />
+    <Setter Property="BorderThickness" Value="{DynamicResource SliderBorderThemeThickness}" />
+    <Setter Property="Foreground" Value="{DynamicResource SliderTrackValueFill}" />
+    <Setter Property="FontFamily" Value="{DynamicResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontSize" Value="{DynamicResource ControlContentThemeFontSize}" />
+    <Setter Property="ManipulationMode" Value="None" />
+    <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+    <Setter Property="FocusVisualMargin" Value="-7,0,-7,0" />
+    <Setter Property="IsFocusEngagementEnabled" Value="True" />
+    <!-- NOTE: This was a contract7Present:Setter, not sure if this is correct -->
+    <Setter Property="CornerRadius" Value="{DynamicResource ControlCornerRadius}" />
```

Comment:
contract7Present:Setter is **[conditional xaml](https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml)** and can be understood something like
if (contract7Present)
    <Setter ... />
With another words, that setter in UWP will be enabled only on Windows 10.0.17763.0 and upper (which supports contract 7). On older OS versions user will not see CornerRadius.

So yeah, this is correct, since Avalonia doesn't depend on Windows version (at least for UI styles).

Reply:
Thanks for the useful info @maxkatz6! That'll help us on another instance of that conditional xaml stuff :)